pixeltable/tests/test_dataframe.py

```diff
@@ -110,3 +112,94 @@ def test_select_literal(self, test_tbl: catalog.Table) -> None:
         t = test_tbl
         res = t.select(1.0).where(t.c2 < 10).show(0)
         assert res.rows == [[1.0]] * 10
+
```

Comment:
missing test cases:
- using the same dataset twice in a row uses the cache
- adding a row to the table invalidates the cached version
- changing the select list invalidates the cached version